File: S1/C#Project/Java/CH12/src/cn/jbit/Drink.java
package cn.jbit;

public class Drink {
	String DrinkName;
	public String getInfo(){
		String name = "";
		if(DrinkName.equals("苹果")){
			name = "甜的";
		}else if(DrinkName.equals("山楂")){
			name = "酸的";
		}else if(DrinkName.equals("柠檬")){
			name = "苦的";
		}
		return name;
	}
}

File: S1/C#作业/2月26日周末作业/Drink.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _2月26日周末作业
{
    class Drink
    {
        public string DrinkName;
        public string name = "";
        public string Taste()
        {
            if (DrinkName.Equals("咖啡"))
            {
                name = "苦";
            }
            else if (DrinkName.Equals("牛奶"))
            {
                name = "甜";
            }
            else if (DrinkName.Equals("柠檬"))
            {
                name = "酸";
            }
            return name;
        }
    }

    class TestDrink
    {
        static void Main(string[] args)
        {
            Drink drink = new Drink();
            Console.WriteLine("输入饮料类型：");
            drink.DrinkName = Console.ReadLine();
            string RinkName = drink.Taste();
            Console.WriteLine(RinkName);
            Console.ReadLine();
        }
    }
}